op-challenger/fault/monitor.go

```diff
@@ -47,7 +49,8 @@ func (m *gameMonitor) progressGames(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to load current block number: %w", err)
 	}
-	games, err := m.source.FetchAllGamesAtBlock(ctx, new(big.Int).SetUint64(blockNum))
+	earliest := time.Now().Unix() - int64(m.gameWindow)
```

Comment:
Very important to not just cast `time.Duration`. Golang may let you, but it's a structured type & has accessor methods for lots of components on it.
```suggestion
	earliest := time.Now().Unix() - int64(m.gameWindow.Seconds())
```

Reply:
That makes sense, thanks.